RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -177,13 +177,16 @@ void resetMaximize()
 	}
 	foreach it in $items[hewn moon-rune spoon, makeshift garbage shirt, broken champagne bottle, snow suit]
 	{
-		if(res != "")
+		if (possessEquipment(it))
 		{
-			res += ",";
+			if(res != "")
+			{
+				res += ",";
+			}
+			// don't want to equip these items automatically
+			// spoon breaks mafia, and the others have limited charges
```

Comment:
Is this still true for spoon? I'm pretty sure i run spoon a lot and it is generally fine. Ill pay attention later today when i play

Reply:
I'm not sure if it is still an issue for mafia. I'll check on their forum. I know the text replacement it does in adventures was breaking all sorts of tracking things in mafia when it was released but they may have a workaround now.

Reply:
I think it was the main source of mafia getting confused about what round it was in combat and either infinite looping or breaking. I see it often equipped during my run, but I am usually doing CS that doesnt do all that much actual adventuring so might not say much.

Reply:
Yeah, looks like there's been no update on the spoon breaking stuff -> https://kolmafia.us/showthread.php?23849-Hewn-moon-rune-spoon/page2
As you say it probably works fine in CS runs but any other run is going to have issues it seems.